packages/sit-onyx/src/components/OnyxListbox/OnyxListbox.vue

```diff
@@ -43,29 +46,49 @@ const slots = defineSlots<{
 const { t } = injectI18n();
 
 /**
- * Currently (visually) active option.
+ * Currently (visually) active/focused option.
```

Comment:
The whole section describing this active handling is called
[Managing focus...](https://www.w3.org/WAI/ARIA/apg/practices/keyboard-interface/#kbd_focus_activedescendant)

I thought it might make it clearer that this is not about "selected", because that was what it sounded like to me.

but if you want I can remove the text changes, I changed the docs on other places like that aswell

Reply:
on the other hand the text describes that what we do is "not moving DOM focus".
Alright, I'll remove those descriptions :)